src/System Application/App/File System/src/Account/FileAccountWizard.Page.al

```diff
@@ -400,15 +400,17 @@ page 9451 "File Account Wizard"
         FeatureTelemetry: Codeunit "Feature Telemetry";
         AccountWasRegistered: Boolean;
         ConnectorSucceeded: Boolean;
+        CTHTok: Label '%1 account has been setup.', Locked = true;
```

Comment:
Tok is used when the label name matches the label value, Like NameTok: Label 'Name'; ... 
These are should use the Lbl postfix